src/pages/drawing/drawing.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div>
+    <div>
+      To start drawing send <code>StartDrawingRequest</code>.
+      <RunExampleButton class="btn btn-primary exampleready" @click="startDrawing">DrawTools.StartDrawingRequest</RunExampleButton>
+      <div>
+        <DocumentationLink type="request" apiDoc="mapping/drawtools/request/startdrawingrequest.md">Documentation for {{ startDrawingRequest }}</DocumentationLink>
+      </div>
```

Comment:
No need to wrap this to div:
```suggestion
        <DocumentationLink type="request" apiDoc="mapping/drawtools/request/startdrawingrequest.md">Documentation for {{ startDrawingRequest }}</DocumentationLink>
```